### ISSUE TITLE
ipsec phase2 cleanup

### ISSUE THREAD

**Issue by zerwes** [Rosa-Luxemburgstiftung-Berlin/ansible-opnsense#21]
ipsec phase2 tunnels removed from config are not deleted in the xml / on the remote, they require a explicit
```yaml
opn_unset:
  - ipsec/phase2[uniqid="ee0e6..."]
```

**Comment by Jonybat** [comment]
How about adding something like this after the ipsecphase2.yml loop?

```
- name: "IPSec ike phase2 search configured uniqid for ikeid {{ ikeid }}"
  delegate_to: localhost
  xml:
    path: "{{ local_config_path }}"
    xpath: "/opnsense/ipsec/phase2[ikeid='{{ ikeid }}']/uniqid"
    content: text
  register: phase2ikeiduniqid_all

- name: "IPSec ike phase2 cleanup for ikeid {{ ikeid }}"
  delegate_to: localhost
  xml:
    path: "{{ local_config_path }}"
    xpath: "/opnsense/ipsec/phase2[ikeid='{{ ikeid }}']/uniqid[text()='{{ item.uniqid }}']"
    state: absent
    pretty_print: yes
  when: item.uniqid not in ipsecphasevar.phase2
  with_items: "{{ phase2ikeiduniqid_all.matches }}"
```

**Comment by zerwes** [comment]
@Jonybat : thanks; yes, looks like a viable solution.
Would you like to open a PR? (would be great if you include a test for it)

**Comment by zerwes** [comment]
fixed in PR #67
